### DISCUSSION TITLE
Changes in Gitbook are not properly reflected, resulting in duplicate README files.

### DISCUSSION THREAD

**Discussion by Benjamin-SCAYLE** [GitbookIO/community#722] [3. Git Sync]
It's a bit challenging to describe this issue, but something doesn't work quite right when renaming or moving pages in Gitbook. Specifically, changes aren't always reflected as expected, which can lead to problems like duplicated README files.

**Example:**

| Group | Page | Path |
|--------|--------|--------|
| September 2024 |  | / |
|   | Release 01.09.  | /readme.md |
| August 2024 |  |  /august-2024/ |
|   | Release 20.08. | /august-2024/Release-20.08.md  | 
|   | Release 01.08. | /august-2024/Release-01.08.md  | 

For example, if a new page (e.g., Release 23.09.) is added above "Release 01.09.," the system may create a "readme (1).md" file in some cases. Moreover, when creating future sections like "October 2024," these duplicated README files are not properly cleaned up, leading to a cluttered and increasingly disorganized file structure.

**Comment by rodrcastro** [comment]
Hey Benjamin, thank you for your message in our community! This is indeed a bit confusing, I'm not sure I fully understand what is happening there. All of the changes you are making (moving pages, renaming, etc) are happening on GitBook, correct?

Do you have an example of a commit where this happened in your repository? Can you please share that at support@gitbook.com? That way we'll be able to ask further questions and understand this issue in more details. I look forward to your feedback!

Thank you,
Rodrigo
